MediaPushPlugin/src/main/java/io/antmedia/plugin/MediaPushPlugin.java

```diff
@@ -92,7 +92,9 @@ public Map<String, RemoteWebDriver> getDrivers() {
 					"--no-sandbox",
 					"--autoplay-policy=no-user-gesture-required",
 					"--disable-background-media-suspend",
+					"--disable-gpu",
 					"--disable-gpu-vsync",
+					"--disable-dev-shm-usage",
```

Comment:
Do we have any drawbacks for "--disable-dev-shm-usage"?

Reply:
Hi @mekya 
Both Selenium and Google Chrome uses shm but in environments like docker container, shared memory is very limited. When we disable the shm usage for Google Chrome, shared memory will continue using by Selenium.

The drawback is because of the Google Chrome will use disk instead of shm, rendering page will be a little bit slow but in the media push plugin, we open a page and record it, I believe it won't effect that much.

Reply:
Hi @mustafaboleken ,
I want to make sure that it does not have an impact on the quality because users expectation is to have high recording. Please make sure its impact with tangible things.

Cheers

Reply:
Hi @mekya 

Sure, my current focus is the preventing the crash and testing it for a while, I will also be sure about the quality.